include/nbl/builtin/hlsl/bxdf/ndf/microfacet_to_light_transform.hlsl

```diff
@@ -23,6 +23,18 @@ enum MicrofacetTransformTypes : uint16_t
    MTT_REFLECT_REFRACT = 0b11
 };
 
+template<typename T>
+struct DualMeasureQuantQuery
+{
+    using scalar_type = T;
+
+    scalar_type getVdotHLdotH() NBL_CONST_MEMBER_FUNC { return VdotHLdotH; }
+    scalar_type getVdotH_etaLdotH() NBL_CONST_MEMBER_FUNC { return VdotH_etaLdotH; }
+
+    scalar_type VdotHLdotH;
+    scalar_type VdotH_etaLdotH;
```

Comment:
the `VdotH_etaLdotH` needs to change (name and what they store) as per https://github.com/Devsh-Graphics-Programming/Nabla/pull/930#discussion_r2413294544

and a comment needs to be left here or in the concept https://github.com/Devsh-Graphics-Programming/Nabla/pull/930#discussion_r2413403654